README.md

```diff
@@ -1,8 +1,10 @@
-[![Build Status](https://travis-ci.org/bespoken/super-simple-unit-testing.svg?branch=master)](https://travis-ci.org/bespoken/super-simple-unit-testing)
-[![codecov](https://codecov.io/gh/bespoken/super-simple-unit-testing/branch/master/graph/badge.svg)](https://codecov.io/gh/bespoken/super-simple-unit-testing)
+[![Build Status](https://travis-ci.org/ryandens/super-simple-unit-testing.svg?branch=master)](https://travis-ci.org/ryandens/super-simple-unit-testing)
```

Comment:
Can you put the changes to the unit tests and travis into their own branch/pull request?